backend/tools/push_backend_framework_to_s3.bash

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Would be good to add a comment what this requires, e.g. the aws cli tool, and how to set the key, etc.

Reply:
Good point, I added a README explaining the purpose of the scripts and how to run them. I did not add documentation on how to actually use `awscli`, as the amazon documentation is very good for that (once you know which commands you want to use) :-).